### PR TITLE
Add contact in list checking method

### PR DESCRIPTION
**Pull request by webarchitect609**
Please, include implementation of 'Check a Contact in a Contact List' method to your package. ( http://documentation.emarsys.com/resource/developers/endpoints/contacts/check-a-contact-in-a-contact-list/ )

### CODE REVIEW

File: src/Snowcap/Emarsys/Client.php [hunk 1]
```diff
@@ -418,6 +418,21 @@ public function getContactsFromContactList($listId, array $data)
     }
 
     /**
+     * Checks whether a specific contact is included in the defined contact list.
+     *
+     * @param integer $contactId
+     * @param integer $listId
+     *
```
Comment (tim-bezhashvyly) [inline src/Snowcap/Emarsys/Client.php]: Please remove this line in order to march coding style of a project.

File: src/Snowcap/Emarsys/Client.php [hunk 2]
```diff
@@ -418,6 +418,21 @@ public function getContactsFromContactList($listId, array $data)
...
+     * @return Response
+     *
+     * @link http://documentation.emarsys.com/resource/developers/endpoints/contacts/check-a-contact-in-a-contact-list/
```
Comment (tim-bezhashvyly) [inline src/Snowcap/Emarsys/Client.php]: Please remove two lines above in order to march coding style of a project.

File: src/Snowcap/Emarsys/Client.php [hunk 3]
```diff
@@ -418,6 +418,21 @@ public function getContactsFromContactList($listId, array $data)
     }
 
     /**
+     * Checks whether a specific contact is included in the defined contact list.
+     *
+     * @param integer $contactId
+     * @param integer $listId
```
Comment (tim-bezhashvyly) [inline src/Snowcap/Emarsys/Client.php]: Please change `integer` into `int` in two lines above in order to march coding style of a project.

File: src/Snowcap/Emarsys/Client.php [hunk 4]
```diff
@@ -418,6 +418,21 @@ public function getContactsFromContactList($listId, array $data)
     }
 
     /**
+     * Checks whether a specific contact is included in the defined contact list.
```
Comment (tim-bezhashvyly) [inline src/Snowcap/Emarsys/Client.php]: Please change "Checks" into "Check" and remove trailing perion in order to march coding style of a project.

Comment (webarchitect609) [inline src/Snowcap/Emarsys/Client.php]: > remove trailing perion

I can't understand you :( What is `perion`?

Comment (webarchitect609) [inline src/Snowcap/Emarsys/Client.php]: > change "Checks" into "Check"
> change integer into int
> remove empty lines around @link and @return

Done. Anything else?

**Comment by webarchitect609** [comment]
Well, `\Snowcap\Emarsys\Client` class has 49 public methods. 41 of them DO HAVE period at the end of description. Also almost every method description sentence is in third person form (Deletes, Creates, Returns, Updates... and etc.). Description is the answer of the question `What does this method do?` and third person form and period at the end is correct according to English Grammar Rules.  

So I fixed my pull-request according to that conclusions to let librarry be clean not only as php but also as English.

**Pull request merged**